_includes/domain-availability/domain-availability-search-input.html

```diff
@@ -0,0 +1,20 @@
+{% comment %}
```

Comment:
hat's the reasoning behind putting this and the availability button in the includes?

Reply:
Rechecked and `check-domain-availability-button.html` is an artifact of a previous conversation when discussing possible options to isolate our search templates that resulted in us deciding that should be in the scope of a separate issue. Since it's not referenced anywhere, will delete `check-domain-availability-button.html` for this PR - thank you for catching that remaining artifact!